konnectd/Makefile

```diff
@@ -84,12 +86,21 @@ lint:
 generate: assets
 	go generate $(GENERATE)
 
-.PHONY: assets
-assets:
+# TODO find a docker container with go and yarn so we can properly build assets in ci
```

Comment:
Might make sense to introduce package caching as well (pre-fetched and bundled in CI container)

Reply:
yes ... but if I ask @micbar about that again he might want to hurt me ... we should schedule it for the next sprint ...

Reply:
why do we need a dedicated container. Just split it in two commands. Run `yarn build` in the nodejs container and go generate in the golang container. That is how we do it in all the other extensions.